stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1823,6 +1824,10 @@ public class ChatClient internal constructor(
                 loggerConfig = ChatLogger.Config(logLevel, loggerHandler),
             )
 
+            if (ToggleService.isInitialized().not()) {
```

Comment:
`MessageListViewModel` can be initialized before this line if called. Pehaps can we wait to the initialilzation in a lazy way using an observable?

Reply:
Nope, this observation doesn't make sense.